sbt-plugin/src/sbt-test/sbt-language-features/blacklist/test

```diff
@@ -0,0 +1,2 @@
+# expects check to fail
+-> languageFeaturesCheck
```

Comment:
It would be good to check the output of the task as well. But I guess the only way to do it would be to write the output of the task to a file. Maybe that’s too complicated…

Reply:
Yes. I think it is a good idea to write output to files so that we can avoid rerun the check on every source file but only on the changed source files. As a byproduct, we can check the output here. Maybe we can do it in another pull request.